src/main/java/com/eternalcode/parcellockers/parcel/ParcelRepository.java

```diff
@@ -1,12 +1,20 @@
 package com.eternalcode.parcellockers.parcel;
 
+import java.util.Optional;
+import java.util.Set;
 import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
 
 public interface ParcelRepository {
 
-    CompletableFuture<Void> update(Parcel parcel);
 
-    CompletableFuture<Parcel> find(UUID parcelUUID);
+    CompletableFuture<Void> save(Parcel parcel);
 
+    CompletableFuture<Optional<Parcel>> findByUuid(UUID uuid);
+
+    CompletableFuture<Set<Parcel>> findAll();
+
+    CompletableFuture<Void> remove(Parcel parcel);
```

Comment:
to co pisałem wyżej na temat ParcelLocker

Reply:
Na razie niech zostanie w celach typu utility

Reply:
jaki ma to cel utility? duplikujesz tylko kod